numpy/__config__.py.in

```diff
@@ -161,4 +161,10 @@ def show(mode=DisplayModes.stdout.value):
             f"Invalid `mode`, use one of: {', '.join([e.value for e in DisplayModes])}"
         )
 
-show.__module__ = "numpy"
+
+def show_config(mode=DisplayModes.stdout.value):
```

Comment:
Not sure if we consider `__config__` public API, so I'll let @rgommers decide.  Maybe this means we should not backport to 2.1.4 at least.
In case the `__module__` thing got backported to that, it may be good to revert it there?

Reply:
So we only need to backport it to 2.2.x

Reply:
We consider `np.show_config` as public API, but not `np.__config__`. I don't immediately spot an API change here that is public.

Reply:
OK, cool, that was the question if `__config__` should be considered truly private or not (either way, some people are using it of course).